.travis.yml

```diff
@@ -1,6 +1,7 @@
 sudo: false
 language: node_js
 before_install:
+  - npm install -g npm@2
```

Comment:
so the version of npm that comes with node 0.8 breaks when it upgrades directly to npm@3, so it has to first upgrade to npm@2 and then from npm@2 upgrade to npm@3